About Cal/content.xml

```diff
@@ -61,7 +61,7 @@
     </page>
     
     <page title="Swift" date="June 2, 2014" icon="swift">
-        <item type="text">I was watching the WWDC 2014 Keynote Livestream. I was shocked to see Apple loosen up to 3rd Party extensions with iOS 8 and I was in love with the new look in OS X Yosemite. But then my jaw hit the floor. With absolutely no warning, Craig Federighi threw the developer world into a frenzy by unveiling Swift. I was watching the keynote with my friend Nate (who will be very jealous of me if I get to go to WWDC). He didn't understand what I was so excited about.</item>
+        <item type="text">I was watching the WWDC 2014 Keynote Livestream. I was shocked to see Apple loosen up to 3rd Party extensions with iOS 8 and I was in love with the new look in OS X Yosemite. But then my jaw hit the floor. With absolutely no warning, Craig Federighi threw the developer world into a frenzy by unveiling Swift. I was watching the keynote with my friend Nate, who didn't understand what I was so excited about.  (He will be very jealous of me if I get to go to WWDC. (We love Craig.))</item>
```

Comment:
Change "(He will be very jealous of me if I get to go to WWDC. (We love Craig.))" to "(He will be very jealous of me if I get to go to WWDC. We love Craig.)"